static/js/components/ReplyButton_test.js

```diff
@@ -0,0 +1,74 @@
+// @flow
+import React from "react"
+import { shallow } from "enzyme"
+import { assert } from "chai"
+
+import ReplyButton from "./ReplyButton"
+import LoginPopup from "./LoginPopup"
+import IntegrationTestHelper from "../util/integration_test_helper"
+import { getCommentReplyInitialValue, replyToCommentKey } from "./CommentForms"
+import { makeComment } from "../factories/comments"
+import { makePost } from "../factories/posts"
+import { shouldIf } from "../lib/test_utils"
+import * as utilFuncs from "../lib/util"
+
+describe("ReplyButton", () => {
+  let helper, beginEditingStub, comment
+
+  beforeEach(() => {
+    helper = new IntegrationTestHelper()
+    beginEditingStub = helper.sandbox.stub()
+    comment = makeComment(makePost())
+  })
+
+  afterEach(() => {
+    helper.cleanup()
+  })
+
+  const renderButton = () =>
+    shallow(
+      <ReplyButton
+        formKey={replyToCommentKey(comment)}
+        initialValue={getCommentReplyInitialValue(comment)}
+        beginEditing={beginEditingStub}
+      />
+    )
```

Comment:
I recently wrote a little function, called `configureShallowRenderer`, which might be helpful here. you call it with a component and a default props object, and it returns a little test renderer - you can use it if you want :)

Reply:
OK